content/v2.0/reference/key-concepts.md

```diff
@@ -0,0 +1,184 @@
+- `bees = 28`
+- `ants = 3`
+
+#### Fields aren't indexed
+
+Fields are required in InfluxDB data and are not indexed. Queries that filter field values must scan all field values to match query conditions. As a result, queries on tags are more performant than queries on fields. Store commonly queried metadata in tags.
+
+#### Tags
+
+The last two columns in the sample data, `location` and `scientist`, are tags.
+Tags include tag keys and tag values that are stored as strings and metadata.
+
+##### Tag keys
+
+The tag keys in the sample data are `location` and `scientist`.
+
+##### Tag values
+
+The tag key `location` has two tag values: `1` and `2`.
+The tag key `scientist` also has two tag values: `anderson` and `mullen`.
+
+##### Tag sets
+
+The collection of tag key-value pairs make up a tag set. The sample data includes the following four tag sets:
+
+- `location = 1`, `scientist = anderson`
+- `location = 2`, `scientist = anderson`
+- `location = 1`, `scientist = mullen`
+- `location = 2`, `scientist = mullen`
+
+#### Tags are indexed
+
+Tags are optional. You don't need tags in your data structure, but it's typically a good idea to include tags.
+Because tags are indexed, queries on tags are faster than queries on fields. This makes tags ideal for storing commonly-queried metadata.
+
+#### Why your schema matters
+
+If most of your queries focus on values in the fields, for example, a query to find when 23 bees were counted:
+
+`SELECT * FROM census WHERE bees = 23`
+
+InfluxDB scans every field value in the dataset for `bees` before the query returns a response. If our sample `census` data grew to millions of rows, to optimize your query, you could rearrange your [schema](/v2.0/reference/glossary/#schema) so the fields (`bees` and `ants`) becomes tags and the tags (`location` and `scientist`) become fields:
+
+| _time                 | _measurement | _field | _value |<span class="tooltip" data-tooltip-text="Tag key">bees</span>|<span class="tooltip" data-tooltip-text="Tag key">ants</span>|
+|:----------------------|--------------|--------|--------|-------|-------|
+| 2019-08-18T00:00:00Z | census |scientist |anderson   |   23 | |
+| <span class="tooltip" data-tooltip-text="Timestamp">2015-08-18T00:00:00Z</span> | <span class="tooltip" data-tooltip-text="Measurement name">census</span> |<span class="tooltip" data-tooltip-text="Field key">scientist</span> | <span class="tooltip" data-tooltip-text="Field value">mullen</span> | <span class="tooltip" data-tooltip-text="Tag value">30</span>  |
+| 2019-08-18T00:06:00Z | census |scientist| anderson| 28 |    |
+|                      |        |         |         |    |    |
+| 2019-08-18T00:00:00Z | census |location | 1       | 23 |    |
+| 2019-08-18T00:00:00Z | census |location | 1       | 30 |    |
+| 2019-08-18T00:06:00Z | census |location | 2       | 28 |    |
+|                      |        |         |         |    |    |
+| 2019-08-18T00:06:00Z | census |location | 2       |    | 3  |
```

Comment:
By restructuring the data, these become two different groups/tables. Here's the updated table(s):

```
| _time                | _measurement | <span class ="tooltip" data-tooltip-text ="Tag key">bees</span> | _field                                                                | _value                                                              |
|:-------------------  |:------------ |:-------                                                         |:--                                                                    |:------                                                              |
| 2019-08-18T00:00:00Z | census       | 23                                                              | location                                                              | loc1                                                                |
| 2019-08-18T00:00:00Z | census       | 23                                                              | scientist                                                             | anderson                                                            |
| 2019-08-18T00:06:00Z | census       | <span class ="tooltip" data-tooltip-text ="Tag value">28</span> | <span class ="tooltip" data-tooltip-text ="Field key">location</span> | <span class ="tooltip" data-tooltip-text ="Field value">loc1</span> |
| 2019-08-18T00:06:00Z | census       | 28                                                              | scientist                                                             | anderson                                                            |

| _time                | _measurement | <span class ="tooltip" data-tooltip-text ="Tag key">ants</span> | _field                                                                | _value                                                              |
|:-------------------  |:------------ |:-------                                                         |:--                                                                    |:------                                                              |
| 2019-08-18T00:00:00Z | census       | 30                                                              | location                                                              | loc2                                                                |
| 2019-08-18T00:00:00Z | census       | 30                                                              | scientist                                                             | mullen                                                              |
| 2019-08-18T00:06:00Z | census       | <span class ="tooltip" data-tooltip-text ="Tag value">32</span> | <span class ="tooltip" data-tooltip-text ="Field key">location</span> | <span class ="tooltip" data-tooltip-text ="Field value">loc2</span> |
| 2019-08-18T00:06:00Z | census       | 32                                                              | scientist                                                             | mullen                                                              |
```